cdap-ui/app/cdap/components/Cloud/Profiles/DetailView/Content/ProfileAssociations/index.js

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {MySearchApi} from 'api/search';
+import {isNilOrEmpty, humanReadableDuration} from 'services/helpers';
+
+require('./ProfileAssociations.scss');
+const HEADERS = [
+  {
+    label: 'Name',
+    property: 'name'
+  },
+  {
+    label: 'Namespace',
+    property: 'namespace'
+  },
+  {
+    label: 'Created',
+    property: 'created'
+  },
+  {
+    label: 'Last 24 hrs runs'
+  },
+  {
+    label: 'Total runs'
+  },
+  {
+    label: 'Last run node/hr'
+  },
+  {
+    label: 'Total node/hr'
+  },
+  {
+    label: 'Schedules'
+  },
+  {
+    label: 'Triggers'
+  }
+];
+
+export default class ProfileAssociations extends Component {
+  static propTypes = {
+    profile: PropTypes.object.isRequired,
+    namespace: PropTypes.string.isRequired
+  };
+
+  state = {
+    associationsMap: {}
+  };
+
+  componentDidMount() {
+    let {namespace, profile} = this.props;
+    MySearchApi
+      .search({
+        namespace: namespace,
```

Comment:
nit: can just do `namespace,`

Also, does this work if `namespace` is `system`? According to these [docs](https://wiki.cask.co/display/CE/CDAP+Provisioning#CDAPProvisioning-APIforProfileMetadataandMetrics), the search endpoint for system level is `/v3/metadata/search`, not `/v3/namespaces/system/metadata/search`

Reply:
For the namespace it is props. This component doesn't use `getCurrentNamespace()`. That API is going away anyway. We need to query `/v3/metadata/search` api for system profiles alone.